DuckDuckGo/FavoritesHomeViewSectionRenderer.swift

```diff
@@ -287,11 +285,10 @@ class FavoritesHomeViewSectionRenderer {
 
         coordinator.drop(dragItem, toItemAt: destinationPath)
 
-        fireActionPixel()
     }
 
     func collectionView(_ collectionView: UICollectionView, itemsForBeginning session: UIDragSession, at indexPath: IndexPath) -> [UIDragItem] {
-        guard let cell = collectionView.cellForItem(at: indexPath) as? FavoriteHomeCell else { return [] }
+        guard let cell = self.collectionView(collectionView, cellForItemAt: indexPath) as? FavoriteHomeCell else { return [] }
```

Comment:
This cannot be reverted - I remember the original code was causing an exception on iOS 18 while reordering favorites.

Reply:
Ah, good spot!

Reply:
Was able to get it crashing instantly and not reverting this fixed that.  Thanks.